ops/terraform/modules/stateless/main.tf

```diff
@@ -205,9 +207,11 @@ module "etl_instance" {
 
   # TODO: Dummy values to get started
   launch_config   = {
-    instance_type = "m5.large" 
-    volume_size   = 100 # GB
-    ami_id        = "ami-0b898040803850657" 
+
+    instance_type = "m5.2xlarge"
+    volume_size   = 100 # GB 
+    ami_id        = var.etl_ami 
+
     key_name      = "bfd-rick-test" 
```

Comment:
s/"bfd-rick-test"/var.ssh_key_name/ ?

Reply:
Yes. It was a place holder SSH key-pair.

Reply:
fixed.